fix just a typo of the word 'word'

shxuy opened the pull request:
<!-- Please use this line to close one or multiple issues when this pullrequest gets merged
You can add another line right under the first one:
resolves #1234
resolves #1235
-->

### This pullrequest changes

<!-- Please describe what your pullrequest is changing -->

alalek commented:
As a "bugfix" this patch should [go](https://github.com/opencv/opencv/wiki/How_to_contribute#faq) into [3.4 branch](https://github.com/opencv/opencv/wiki/Branches) first. We will merge changes from 3.4 into master regularly (weekly/bi-weekly).

So, please:
- change "base" branch of this PR: master => 3.4 (use "Edit" button near PR title)
- rebase your commits from master onto 3.4 branch. For example:
    `git rebase -i --onto upstream/3.4 upstream/master`
    (check list of your commits, save and quit (Esc + "wq" + Enter)
    where `upstream` is configured by following [this GitHub guide](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and fetched (`git fetch upstream`).
- push rebased commits into source branch of your fork (with `--force` option)

Note: no needs to re-open PR, apply changes "inplace".

shxuy commented:
Thanks. I have changed "base" branch of this PR. Since I just modified the document on Github and I have not downloaded the repository to my own computer and I have not configured the upstream , it seems that I should not worry about pushing rebased commits into source branch of my fork.

shxuy commented:
By the way, I found another tiny problem of Python tutorial which puzzled me and raised a question in the [forum](http://answers.opencv.org/question/211391/why-is-the-order-of-color-channels-in-funtion-copymakeborder-in-python-version-not-b-g-r-but-r-g-b/). The problem is due to the change of matplotlib. I think maybe we have to modify the code example on the page and I am sure how to fix it.
In my opinion, in order to work with the new version of matplotlib, perhaps we have to reverse the order of color channels of every images which is time-consuming but I do not have a better idea. Here is my idea:
> import cv2 as cv
from matplotlib import pyplot as plt
BLUE = [255,0,0]
img1 = cv.imread('opencv-logo.png')
replicate = cv.copyMakeBorder(img1,10,10,10,10,cv.BORDER_REPLICATE)[...,::-1]
reflect = cv.copyMakeBorder(img1,10,10,10,10,cv.BORDER_REFLECT)[...,::-1]
reflect101 = cv.copyMakeBorder(img1,10,10,10,10,cv.BORDER_REFLECT_101)[...,::-1]
wrap = cv.copyMakeBorder(img1,10,10,10,10,cv.BORDER_WRAP)[...,::-1]
constant = cv.copyMakeBorder(img1,10,10,10,10,cv.BORDER_CONSTANT,value=BLUE)[...,::-1]
plt.subplot(231),plt.imshow(img1,'gray'),plt.title('ORIGINAL'),plt.xticks([]),plt.yticks([])
plt.subplot(232),plt.imshow(replicate,'gray'),plt.title('REPLICATE'),plt.xticks([]),plt.yticks([])
plt.subplot(233),plt.imshow(reflect,'gray'),plt.title('REFLECT'),plt.xticks([]),plt.yticks([])
plt.subplot(234),plt.imshow(reflect101,'gray'),plt.title('REFLECT_101'),plt.xticks([]),plt.yticks([])
plt.subplot(235),plt.imshow(wrap,'gray'),plt.title('WRAP'),plt.xticks([]),plt.yticks([])
plt.subplot(236),plt.imshow(constant,'gray'),plt.title('CONSTANT'),plt.xticks([]),plt.yticks([])
plt.show()

Could you fix it for the perspective learners? Thanks.
